test/extended/security/scc.go

```diff
@@ -30,7 +30,7 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
 	oc := exutil.NewCLIWithPodSecurityLevel("scc", admissionapi.LevelPrivileged)
 	ctx := context.Background()
 
-	g.It("TestPodUpdateSCCEnforcement", func() {
+	g.It("TestPodUpdateSCCEnforcement [apigroup:user.openshift.io][apigroup:authorization.openshift.io]", func() {
```

Comment:
This is going to bite us.  We'll need to fix these too to ensure SCC functions.  We'll need a way to handle different users.  Perhaps serviceaccounts instead of regular users.